tasks/main.yaml

```diff
@@ -34,7 +34,19 @@
     ansible_distribution_major_version | int <= 6 and
     grub_cmdline_remove_args | length > 0
 
-- name: Update Grub default config for RedHat >= 7
+- name: Check fips cmdline
+  set_fact:
+    grub_cmdline_add_args: "{{ grub_cmdline_add_args }} + ['fips=1']"
+  when: >
+    grub_enable_fips and
+    "'fips=1' not in grub_cmdline_add_args | join(' ')"
+
+- name: Install dracut-fips
+  package:
+    name: dracut-fips
```

Comment:
Please could you make the package name as a variable? Like that a specific version could be installed.